.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,10 @@
 
 ## What is the value of this and can you measure success?
 
+<!--
+If setting up an AB test, make sure you have read our [AB testing doc](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md).
```

Comment:
> Do you mean able?

No, though I agree my phrasing is confusing. I meant that in general, I would expect people who have set up an AB test to have read that doc at least once. But the way you would read that doc when trying to figure out how to create a working AB test would be different from the way you would read it to make sure you've done everything right. And a little paragraph is easy to miss even if you do a due diligence sweep of the doc afterwards.

> Maybe we can just make the first part of this better

That's a good idea. Maybe there's just an extra step after "Running the test" which is "Checking for test clashes"?